scripts/emulator-testing/dataconnect-test-runner.ts

```diff
@@ -38,7 +39,7 @@ async function run(): Promise<void> {
     await emulator.setUp();
     await runTest(emulator.port);
   } finally {
-    await emulator.tearDown();
```

Comment:
Why was this change made?

Reply:
It looks like firestore and database also await this expression. We should probably remove await from there too.